async_processor/worker.py

```diff
@@ -29,13 +28,11 @@ def __init__(self, worker_config: WorkerConfig, processor: Processor):
         self._output = worker_config.output_config.to_output()
         self._processor = processor
         self._healthy = True
-        signal.signal(signal.SIGTERM, self.stop)
```

Comment:
why did we remove this

Reply:
FastAPI lifespan will handle it.

Reply:
https://github.com/truefoundry/async_service/pull/11/files#diff-97e48d2f2456d5b71489c9e2fb6292375ce696a38437e15108c24a54c351c3e1R94

Reply:
https://fastapi.tiangolo.com/advanced/events/